src/Entity/EntityId.php

```diff
@@ -68,7 +80,11 @@ public function getSerialization() {
 	public static function splitSerialization( $serialization ) {
 		self::assertValidSerialization( $serialization );
 
-		$parts = explode( ':', self::normalizeIdSerialization( $serialization ) );
+		return self::getSerializationParts( self::normalizeIdSerialization( $serialization ) );
+	}
+
+	private static function getSerializationParts( $serialization ) {
```

Comment:
The name sounds like a getter, which this is not. "splitSerialization" would be descriptive, but it's take. splitSerializationInternal is ugly, but ok for a private method, I guess.

Alternatively, don't add this method at all. Why can't we just use splitSerialization?

Reply:
So it was introduced to be able to chop serialization into pieces but not run validation checks.